### DISCUSSION TITLE
Episode sort order:  How to use?

### DISCUSSION THREAD

**Discussion by lastboat36** [PodcastGenerator/PodcastGenerator#713] [Q&A] [answered]
I have checked box in Generator Configuration.  Will it change order to edited existing podcasts?  I don't see it working.  Any ideas how to arrange podcasts?  thanks

**System Information**
Podcast Generator Version: 
Webserver: 
PHP Version:

**Podcast Generator Config**

```
INSERT THE **CENSORED** CONFIG HERE
``` 

**Description of your problem:**

**Comment by charabaruk** [comment, marked as answer]
This feature doesn't make any changes to episodes themselves, but rather changes how episodes are sorted for display and the RSS feed. By default ("Timestamp"), episodes are sorted in reverse chronological order (newest episodes first) but "Season and episode number" uses those values for episodes, when available, to sort the episodes. If you don't use the season and episode fields, you'll see no change in episode order as episodes without those values continue to be sorted in reverse chronological order.
